### PR TITLE
Investigation Insights Workbook

### PR DESCRIPTION
**Pull request by briandelmsft**
Adding new Investigation Insights Workbook

### CODE REVIEW

File: Workbooks/WorkbooksMetadata.json [hunk 1]
```diff
@@ -514,7 +514,7 @@
     "dataTypesDependencies": [ "CommonSecurityLog", "SecurityEvent", "Syslog" ],
     "dataConnectorsDependencies": [],
     "previewImagesFileNames": [ "AzureSentinelSecurityStatusBlack.png", "AzureSentinelSecurityStatusWhite.png" ],
-    "version": "1.2",
+    "version": "1.3",
```
Comment (morshabi) [inline Workbooks/WorkbooksMetadata.json:517]: Why did you update this version I can't see changes in the SecurityStatus workbook json itself

File: Workbooks/WorkbooksMetadata.json [hunk 2]
```diff
@@ -657,7 +657,7 @@
     "dataTypesDependencies": [ "SecurityAlert", "ProtectionStatus", "SecurityRecommendation", "SecurityBaseline", "SecurityBaselineSummary", "Update", "ConfigurationChange" ],
     "dataConnectorsDependencies": [ "AzureSecurityCenter" ],
     "previewImagesFileNames": [ "ASCCaPBlack.png", "ASCCaPWhite.png" ],
-    "version": "1.1",
+    "version": "1.2",
```
Comment (morshabi) [inline Workbooks/WorkbooksMetadata.json:660]: Why did you update this version I can't see changes in the Workspace audit workbook json itself?
the same for the provider field

File: Workbooks/WorkbooksMetadata.json [hunk 3]
```diff
@@ -956,19 +956,32 @@
...
   "workbookKey": "MITREATTACKWorkbook",
   "logoFileName": "Azure_Sentinel.svg",
   "description": "Workbook to showcase MITRE ATT&CK Coverage for Azure Sentinel",
   "dataTypesDependencies": [],
   "dataConnectorsDependencies": [],
-  "previewImagesFileNames": [ "WorkspaceAuditingWhite.png", "WorkspaceAuditingBlack.png" ],
+  "previewImagesFileNames": [ "MITREATTACKWhite1.png", "MITREATTACKWhite2.png", "MITREATTACKBlack1.png", "MITREATTACKBlack2.png" ],
   "version": "1.0",
-  "title": "Workspace audit",
+  "title": "MITRE ATT&CK Workbook",
```
Comment (morshabi) [inline Workbooks/WorkbooksMetadata.json:969]: Is this a planned change or just a conflict?

File: Workbooks/WorkbooksMetadata.json (hunk 1 above)
Comment (briandelmsft) [reply Workbooks/WorkbooksMetadata.json]: i was trying to fix the conflict that was created by other accepted PRs.  I have reverted this commit

File: Workbooks/WorkbooksMetadata.json (hunk 2 above)
Comment (briandelmsft) [reply Workbooks/WorkbooksMetadata.json]: i was trying to fix the conflict that was created by other accepted PRs.  I have reverted this commit

File: Workbooks/WorkbooksMetadata.json (hunk 3 above)
Comment (briandelmsft) [reply Workbooks/WorkbooksMetadata.json]: this was created by a conflict that i was trying to fix in the other 2 commits, would you prefer if i close this one and submit a new PR?

**Comment by briandelmsft** [comment]
I created a new pull request, PR #1186 that does not have this conflict.  Closing this PR.

**Pull request closed without merging**